src/sql/workbench/contrib/editData/browser/editDataResultsEditor.ts

```diff
@@ -94,7 +94,7 @@ export class EditDataResultsEditor extends BaseEditor {
 	 */
 	private _bootstrapAngular(): void {
 		let input = <EditDataResultsInput>this.input;
-		let uri = input.uri;
+		let uri = unescape(input.uri);
```

Comment:
Why aren't you unescaping it in the EditDataResultsInput like you did for EditDataInput?